style.css

```diff
@@ -0,0 +1,4 @@
+/* stylelint-disable */
```

Comment:
- Enable your linters and fix issues.

Reply:
Thanks @Carlos for taking your time to review my project.I have worked on the requested changes!